### ISSUE TITLE
[Bug]: RAGE MP

### ISSUE THREAD

**Issue by JaredHuh** [QuintenQVD0/Q_eggs#37]
### Panel Version

1.11.3

### Wings Version

1.11.4

### Service

RAGE MP

### Modified

No, I did not modify the egg

### Expected Behavior

Install the server and start it successfully

### Actual Behavior

It starts, but then an error pops up.

logs:
https://pastebin.com/yUyeBuHB

### Steps To Reproduce

- Install egg
- Install server
- Try to start it

### Install logs

https://pteropaste.com/ak41cnd86rmi/

**Comment by QuintenQVD0** [comment]
Wait what? How did ir not work. I have a look

**Comment by QuintenQVD0** [comment]
my logs

```bash
container@pterodactyl~ AMD64 Emualtor image box64
Dynarec for ARM64, with extension: ASIMD CRC32 PageSize:4096 Cores:4
Params database has 15 entries
Box64 with Dynarec v0.2.1 2e59b56f built on Feb 19 2023 07:05:23
:/home/container$ $(if [ "${ARCH}" == "arm64" ]; then echo -n "box64 ./ragemp-server"; else echo -n "./ragemp-server"; fi)
Dynarec for ARM64, with extension: ASIMD CRC32 PageSize:4096 Cores:4
Params database has 15 entries
Box64 with Dynarec v0.2.1 2e59b56f built on Feb 19 2023 07:05:23
Using default BOX64_LD_LIBRARY_PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/
Using default BOX64_PATH: ./:bin/
Counted 24 Env var
Looking for ./ragemp-server
Rename process to "ragemp-server"
Using native(wrapped) libdl.so.2
Using native(wrapped) libc.so.6
Using native(wrapped) ld-linux-x86-64.so.2
Using native(wrapped) libpthread.so.0
Using native(wrapped) libutil.so.1
Using native(wrapped) librt.so.1
Using native(wrapped) libm.so.6
Using emulated /usr/lib/x86_64-linux-gnu/libstdc++.so.6
Using emulated /usr/lib/x86_64-linux-gnu/libgcc_s.so.1
[INFO] Starting RAGE Multiplayer server...

[============================================================]
||
||      MaxPlayers              50
||      Sync rate               40ms
||      Name                    RAGE:MP Unofficial server
||      Gamemode                freeroam
||      Streaming distance      500
||      Announcement            disabled
||      Voice chat              disabled
||      Address                 0.0.0.0:25568
||      Connection limits       disabled
||      Encryption              enabled
||      NodeJS                  enabled
||      C#                      disabled
||
[============================================================]

[INFO] Loading NodeJS packages...
[INFO] Starting packages...
[DONE] Server packages have been started.
[DONE] Started resource transfer server at 25569 port.
[DONE] Client-side packages weight: 0.000000 MB (uncompressed: 0.000000 MB).
[INFO] Initializing networking...
[DONE] Networking has been started: (IPv4-only) at 127.0.0.1:25568
[DONE] The server is ready to accept connections.
container@pterodactyl~ Server marked as running...
container@pterodactyl~ Server marked as stopping...
```

**Comment by JaredHuh** [comment]
> my logs
> 
> ```shell
> container@pterodactyl~ AMD64 Emualtor image box64
> Dynarec for ARM64, with extension: ASIMD CRC32 PageSize:4096 Cores:4
> Params database has 15 entries
> Box64 with Dynarec v0.2.1 2e59b56f built on Feb 19 2023 07:05:23
> :/home/container$ $(if [ "${ARCH}" == "arm64" ]; then echo -n "box64 ./ragemp-server"; else echo -n "./ragemp-server"; fi)
> Dynarec for ARM64, with extension: ASIMD CRC32 PageSize:4096 Cores:4
> Params database has 15 entries
> Box64 with Dynarec v0.2.1 2e59b56f built on Feb 19 2023 07:05:23
> Using default BOX64_LD_LIBRARY_PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/
> Using default BOX64_PATH: ./:bin/
> Counted 24 Env var
> Looking for ./ragemp-server
> Rename process to "ragemp-server"
> Using native(wrapped) libdl.so.2
> Using native(wrapped) libc.so.6
> Using native(wrapped) ld-linux-x86-64.so.2
> Using native(wrapped) libpthread.so.0
> Using native(wrapped) libutil.so.1
> Using native(wrapped) librt.so.1
> Using native(wrapped) libm.so.6
> Using emulated /usr/lib/x86_64-linux-gnu/libstdc++.so.6
> Using emulated /usr/lib/x86_64-linux-gnu/libgcc_s.so.1
> [INFO] Starting RAGE Multiplayer server...
> 
> [============================================================]
> ||
> ||      MaxPlayers              50
> ||      Sync rate               40ms
> ||      Name                    RAGE:MP Unofficial server
> ||      Gamemode                freeroam
> ||      Streaming distance      500
> ||      Announcement            disabled
> ||      Voice chat              disabled
> ||      Address                 0.0.0.0:25568
> ||      Connection limits       disabled
> ||      Encryption              enabled
> ||      NodeJS                  enabled
> ||      C#                      disabled
> ||
> [============================================================]
> 
> [INFO] Loading NodeJS packages...
> [INFO] Starting packages...
> [DONE] Server packages have been started.
> [DONE] Started resource transfer server at 25569 port.
> [DONE] Client-side packages weight: 0.000000 MB (uncompressed: 0.000000 MB).
> [INFO] Initializing networking...
> [DONE] Networking has been started: (IPv4-only) at 127.0.0.1:25568
> [DONE] The server is ready to accept connections.
> container@pterodactyl~ Server marked as running...
> container@pterodactyl~ Server marked as stopping...
> ```

Hmmm... I'll try to find issue on my side, I have some ideas.

**Comment by QuintenQVD0** [comment]
> Are you using the same egg? I have "AES", "PMULL" and "ATOMICS" dynarec extensions, which you do not have in your logs.

Yes, my egg is the onz out of the repo. The extentions are based on kernel and hidt device. Mine is a pi 4. Only difrence is that i also have box64 on the host os. But that should not be needed

**Comment by QuintenQVD0** [comment]
You have more so that should not be an issue. How old is your kernel?

**Comment by JaredHuh** [comment]
> You have more so that should not be an issue. How old is your kernel?

5.15.0-1029-oracle

**Comment by JaredHuh** [comment]
Ok, I have no idea what's wrong...

**Comment by QuintenQVD0** [comment]
Same for me, it should work. And it does work for me.

**Comment by QuintenQVD0** [comment]
the rror you have only show up for me if you press stop
